src/makeCalculator.js

```diff
@@ -5,6 +5,38 @@
  */
 function makeCalculator() {
   // write code here
+  return {
+    result: 0,
+
+    add(number) {
+      this.result += number;
+    },
+    subtract(number) {
+      this.result -= number;
+    },
+    multiply(number) {
+      this.result *= number;
+    },
+    divide(number) {
+      if (number === 0) {
+        throw new Error('Error');
```

Comment:
The error message thrown when dividing by zero should be more descriptive to convey the specific error to the user. Instead of a generic 'Error', consider specifying the nature of the error, such as 'Division by zero is not allowed'.